src/en/news/blog/2025/user-accounts-dashboard/index.md

```diff
@@ -0,0 +1,155 @@
+    *Figure: Step 2.1 - Enter account details*
+
+![Step 2.2: Create Account: Enter quota details](images/enter-account-form-details2.png)  
+    *Figure: Step 2.2 - Enter quota details*
+
+After entering the details click on create Account button, post successful account creation you will be redirected to the accounts list page in which we can see the newly created account listed in table, and on expanding the table row, we can see the account details which shows the Account Quota & Bucket Quota for the account.
+
+![Step 3: Account list & details](images/account-details.png)  
+    *Figure: Step 3 - Account details*
+
+### 3.2 Link user with an Account
+
+After creating an account, the next step would be to create a account user. Lets see how its done.
+
+Navigate to the user page and click on `Create` button for creating a user.
+
+An account user can be created by two ways:  
+ - Creating a new user and linking it with an account at the same time.
+ - If you already have a user created then that user can be edited, and can be linked to an account.
+
+You will be seeing the user form in which `Link Account` will be the first input selector from which you can choose to link the user with an account, from the account list.
+
+> **NOTE:** This option would be visible only if you have accounts created.  
+
+![Step 4.1: Create User: Link user with account](images/link-user-with-account.png)  
+*Figure: Step 4.1 - Link user with account*
+
+Also, as seen once the account is selected, there will be another option to make it as a `root account user`. We can select the checkbox if this user needs to have additional permission to manage/create other user.
+
+#### Apply Managed Policies to account user
+
+If the account user is non-root, there will be another option that will be visible which is **Manage Policies**.
+These policies give broad permissions for users:
+ - Full access versions (e.g. AmazonS3FullAccess) grant complete access to the service.
+ - Read-only versions limit actions to safe, non‑modifying operations.
+
+We can select the option based on the permission/access to be given.
+
+![Step 4.2: Create User: Apply managed policy](images/attach-managed-policy.png)  
+*Figure: Step 4.2 - Apply managed policy to account user*  
+
+After filling other required details we can click on create/edit user based on scenarios described above. Once the request completes you will be navigated back to user list page.
+
+Here, we can check the details of the user linked with an account, in the table `Account name` shows the user is linked with an account, and also if user is a root account user then it is depicted with a icon with username as seen below.  
+
+![Step 4.3: User Details: Account root user](images/account-root-user.png)  
+*Figure: Step 4.3 - Account root user*  
+
+We can also see more details of that user by going to user details section by expanding the table row.
```

Comment:
You can view more details about the user by expanding the table row to access the User Details section.